### PR TITLE
Enable workflow button when workflows available

### LINKED ISSUE #5001: Bug in "Run workflow"

### PR DESCRIPTION
**Pull request by andreas-el**
**Issue**
Resolves #5001 

**Approach**
_Short description of the approach_

## Pre review checklist

- [x] Added appropriate release note label
- [x] PR title captures the intent of the changes, and is fitting for release notes.
- [x] Commit history is consistent and clean, in line with the [contribution guidelines](https://github.com/equinor/ert/blob/main/CONTRIBUTING.md).
- [x] Updated documentation
- [x] Ensured new behaviour is tested

Adding labels helps the maintainers when writing release notes. This is the [list of release note labels](https://github.com/equinor/ert/labels?q=release-notes).

### CODE REVIEW

File: src/ert/gui/tools/workflows/workflows_tool.py [hunk 1]
```diff
@@ -8,7 +8,7 @@ class WorkflowsTool(Tool):
     def __init__(self, ert, notifier):
         self.notifier = notifier
         self.ert = ert
-        enabled = len(ert.resConfig().workflow_jobs) > 0
+        enabled = len(ert.resConfig().workflows.keys()) > 0
```
Comment (DanSava) [inline src/ert/gui/tools/workflows/workflows_tool.py:11]: I think this is the same thing
```suggestion
        enabled = len(ert.resConfig().workflows) > 0
```

**Review by DanSava — APPROVED** [review]
Looks good just one small suggestion

File: src/ert/gui/tools/workflows/workflows_tool.py (hunk 1 above)
Comment (andreas-el) [reply src/ert/gui/tools/workflows/workflows_tool.py]: Seems like the proper thing to do. :+1:

**Comment by andreas-el** [comment]
Jenkins retest this please!

**Comment by oysteoh** [comment]
Jenkins: Test this please!

**Pull request merged**